docs/articles/toc.yml

```diff
@@ -23,6 +23,10 @@
     href: beyond_basics/sharding.md
   - name: Message Builder
     href: beyond_basics/messagebuilder.md
+- name: Advanced Topics
+  items:
+  - name: Buttons
+    href: advanced_topics/buttons.md
```

Comment:
This part should be after Interactivity imo.

Reply:
Oh wait oops I may have screwed up the order. Yeah.